src/main/java/io/supertokens/webserver/api/multitenancy/DisassociateUserFromTenant.java

```diff
@@ -79,7 +82,10 @@ protected void doPost(HttpServletRequest req, HttpServletResponse resp) throws I
             JsonObject result = new JsonObject();
             result.addProperty("status", "UNKNOWN_USER_ID_ERROR");
             super.sendJsonResponse(200, result, resp);
-
+        } catch (DisassociationNotAllowedException e) {
+            JsonObject result = new JsonObject();
+            result.addProperty("status", "DISASSOCIATION_NOT_ALLOWED_ERROR");
+            super.sendJsonResponse(200, result, resp);
```

Comment:
add a reason string, and also have this showed on the backend sdk

Reply:
Done